### PR TITLE
Docblock/tribe is ajax view request

### PR DESCRIPTION
**Pull request by deblynprado**
### 🎫 Ticket

[TICKET_ID]
No Ticket ID - Regular Support Engineers task

### 🗒️ Description

Updating Docblock section

### ✔️ Checklist
- [x] Ran `npm run changelog` to add changelog file(s). More info [here](https://docs.theeventscalendar.com/developer/git/changelogs/#process)
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

### CODE REVIEW

File: src/functions/template-tags/general.php [hunk 1]
```diff
@@ -252,14 +252,15 @@ function tribe_get_template_part( $slug, $name = null, array $data = null ) {
 
 if ( ! function_exists( 'tribe_is_ajax_view_request' ) ) {
 	/**
-	 * Check if the current request is for a tribe view via ajax
+	 * Determines if the current request is for a Tribe Events view via AJAX.
```
Comment (codingmusician) [inline src/functions/template-tags/general.php:255]: I think we can remove the reference to `Tribe` here. It's been awhile. ;-)
```suggestion
	 * Determines if the current request is for an Events view via AJAX.
```

File: src/functions/template-tags/general.php [hunk 2]
```diff
@@ -283,6 +284,14 @@ function tribe_is_ajax_view_request( $view = false ) {
 			}
 		}
 
+		/**
+		 * Filters whether the current request is an AJAX request for a specific Tribe view.
+		 *
+		 * @since 6.0.0
+		 *
+		 * @param bool       $is_ajax_view_request Whether the request is an AJAX request for the specified view.
+		 * @param bool|string $view                 The view slug being checked.
```
Comment (codingmusician) [inline src/functions/template-tags/general.php:293]:
```suggestion
		 * @param bool        $is_ajax_view_request Whether the request is an AJAX request for the specified view.
		 * @param bool|string $view                 The view slug being checked.
```

File: src/functions/template-tags/general.php (hunk 1 above)
Comment (deblynprado) [reply src/functions/template-tags/general.php]: Good point! Already fixed :D

File: src/functions/template-tags/general.php (hunk 2 above)
Comment (deblynprado) [reply src/functions/template-tags/general.php]: Há! Eagle Eyes!

**Review by codingmusician — APPROVED** [review]

**Pull request closed without merging**